ajax/libs/mmenu-js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "mmenu-js",
+  "filename": "mmenu.min.js",
+  "author": "Fred Heusschen <info@frebsite.nl>",
+  "license": "CC-BY-NC-4.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FrDH/mmenu-js.git"
+  },
+  "description": "The best javascript plugin for app look-alike on- and off-canvas menus with sliding submenus for your website and webapp.",
+  "keywords": [
+    "app",
+    "list",
+    "listview",
+    "megamenu",
+    "menu",
+    "mmenu",
+    "mobile",
+    "navigation",
+    "off-canvas",
+    "on-canvas",
+    "curtain",
+    "panels",
+    "submenu"
+  ],
+  "npmName": "mmenu-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
See above tree, are all these files required for the library, or do we only need certain ones from within `dist`?

(This is a question, not a required change)